workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGenerationLib/Tests/Model/NameClashingTests.cs

```diff
@@ -0,0 +1,80 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using Improbable.Gdk.CodeGeneration.Model;
+using Improbable.Gdk.CodeGeneration.Model.Details;
+using NUnit.Framework;
+
+namespace Improbable.Gdk.CodeGeneration.Tests.Model
+{
+    [TestFixture]
+    public class NameClashingTests
```

Comment:
Can we also assert that we are logging errors/warnings (whichever it is) to the user?

Reply:
I think there's a way, I'll try it out

Reply:
ye it works